datashader/glyphs.py

```diff
@@ -123,7 +123,8 @@ def __init__(self, x, y, z=None, weight_type=True, interp=True):
 
     @property
     def inputs(self):
-        return tuple([self.x, self.y] + list(self.z))
+        return (tuple([self.x, self.y] + list(self.z)) +
+                (self.weight_type, self.interpolate))
```

Comment:
This change was needed because the `inputs` tuple is used by the parent class to implement hashing and equality, which are in tern used for memoization.  Without this, I was seeing cases where repeated use of `canvas.trimesh` with different values for `interpolate` was not resulting in updated aggregation behavior.